src/lifecycle/render.js

```diff
@@ -1,18 +1,39 @@
 import { patchInner } from 'incremental-dom';
-import { shadowRoot as $shadowRoot } from '../util/symbols';
+import { connected as $connected, shadowRoot as $shadowRoot, state as $state } from '../util/symbols';
```

Comment:
Could / should we export the symbols with the `$` prefix? Would this be another breaking change?